### PR TITLE
Update to bazel 0.25.1

### PR DESCRIPTION
**Pull request by pmuetschard**

### CODE REVIEW

**Comment by hevrard** [comment]
Linux and Windows builds still hit bazel errors apparently.

**Comment by hevrard** [comment]
It's all green expect on Windows:
```
ERROR: C:/users/builder/_bazel_kbuilder/fsjplg7y/external/gapid/tools/build/rules/repository.bzl:100:9: no such package '@androidsdk//': C:\users\builder\_bazel_kbuilder\fsjplg7y\external\androidsdk and referenced by '//external:android/dx_jar_import'
ERROR: Analysis of target '//:pkg' failed; build aborted: Analysis failed
```
I can't spot reference to 'androidsdk' in your changes, so maybe it's just Kokoro being flaky?

I'll start a new kokoro build to see if it goes through.

File: BUILDING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Building GAPID
 
-GAPID uses the [Bazel build system](https://bazel.build/). The current minimum required version of bazel is 0.18.1.
+GAPID uses the [Bazel build system](https://bazel.build/). The current minimum required version of bazel is 0.25.1.
```
Comment (AWoloszyn) [inline BUILDING.md]: Gross big jump. But I assume its basically impossible to avoid.

Comment (pmuetschard) [reply BUILDING.md]: The reality is that we were building with 0.20 before, but yeah, we should keep up with it a bit more.

**Pull request merged**